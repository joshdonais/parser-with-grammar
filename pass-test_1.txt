program abc
dec
i1,i2:int;
x,y:real
begin
	Read(i2,x);
	y=i2*x+2;
	Write(y)
end.
