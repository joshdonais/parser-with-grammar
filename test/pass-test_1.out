PROGSYM
IDENT
DECSYM
IDENT
COMMA
IDENT
COLN
IDENT
SEMICOLN
IDENT
COMMA
IDENT
COLN
IDENT
BEGINSYM
READSYM
LPAREN
IDENT
COMMA
IDENT
RPAREN
SEMICOLN
IDENT
ASSIGN
IDENT
MULTIPLY
IDENT
ADD
NUM
SEMICOLN
WRITESYM
LPAREN
IDENT
RPAREN
ENDSYM
